GET_STARTED.md

```diff
@@ -0,0 +1,390 @@
+ ...
+ # CCF Network initialization needed before the interaction with the service
+```
+
+##### CCF Node Configuration file
+The configuration for each CCF node must be contained in a single JSON configuration file like [cchost_config_enclave_js.json - cchost_config_virtual_js.json], [ CCF node config file documentation](https://microsoft.github.io/CCF/main/operations/configuration.html)
+
+##### CCF network initialization
+After the container run, a network is started with one (node - member), you need to execute the following governance steps to initialize the network, [check Network governance section](#network-governance)
+- Activate the network members (to begin network governance)
+- Add users (using proposal)
+- Deploy the application (using proposal)
+- Open the network for users (using proposal)
+
+#### Testing: Using Linux Machine
+To Start a test CCF network on a VM, it requires [CCF to be intalled](https://microsoft.github.io/CCF/main/build_apps/install_bin.html)
+
+Start the CCF network using the cchost in release mode
+```bash
+ /opt/ccf/bin/cchost --config ./config/cchost_config_enclave_js.json
+```
+Or virtual mode
+```bash
+/opt/ccf/bin/cchost --config ./config/cchost_config_virtual_js.json
+```
+
+##### CCF Node Configuration file
+The configuration for each CCF node must be contained in a single JSON configuration file like [cchost_config_enclave_js.json - cchost_config_virtual_js.json], [ CCF node config file documentation](https://microsoft.github.io/CCF/main/operations/configuration.html)
+
+##### CCF network initialization
+Network is started with one (node - member), you need to execute the following governance steps to initialize the network,, [check Network governance section](#network-governance)
+- Activate the network members (to start a network governance)
+- Add users (using proposal)
+- Deploy the application (using proposal)
+- Open the network for users (using proposal)
+
+### Testing: Application Endpoints  
+
+Now you can send requests to your endpoints, the following sample requests to log application [(ccf-app-template)](https://github.com/microsoft/ccf-app-template/tree/main/js)
+
+In another terminal:
+
+#### Log application
+```bash
+ # send log message to be saved
+ curl -X POST https://127.0.0.1:8000/app/log?id=1 --cacert ./workspace/sandbox_common/service_cert.pem -H "Content-Type: application/json" --data '{"msg": "hello world"}'
+ # retrieve log message
+ curl https://127.0.0.1:8000/app/log?id=1 --cacert ./workspace/sandbox_common/service_cert.pem
+ # return:> hello world
+```
+
+#### Banking application
+```bash
+echo "Define vars"
+user0_id=$(openssl x509 -in "user0_cert.pem" -noout -fingerprint -sha256 | cut -d "=" -f 2 | sed 's/://g' | awk '{print tolower($0)}')
+user1_id=$(openssl x509 -in "user1_cert.pem" -noout -fingerprint -sha256 | cut -d "=" -f 2 | sed 's/://g' | awk '{print tolower($0)}')
+account_type0='current_account'
+account_type1='savings_account'
```

Comment:
The commands related to the users in the banking-app assumes we already have all the certificates/keys generated. 
Are we covering this on this guide? Maybe it's worth mentioning that there's a script for creating them.